### PR TITLE
chartvalues: add credentiald

### PR DESCRIPTION
**Pull request by kopiczko**
Towards https://github.com/giantswarm/giantswarm/issues/6736.

Requires https://github.com/giantswarm/credentiald/pull/34.

### CODE REVIEW

File: pkg/chartvalues/credentiald.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	Azure              CredentialdConfigAzure
+	Deployment         CredentialdConfigDeployment
+	RegistryPullSecret string
+}
+
+type CredentialdConfigAWS struct {
+	CredentialDefault CredentialdConfigAWSCredentialDefault
+}
+
+type CredentialdConfigAWSCredentialDefault struct {
```
Comment (tfussell) [inline pkg/chartvalues/credentiald.go:21]: This isn't a criticism, just learning more about how this works. Why isn't `AdminARN` needed here? https://github.com/giantswarm/e2etemplates/blob/master/pkg/chartvalues/aws_operator.go#L40  Also, should `CredentialDefault` be removed from the `aws_operator` `chartvalues`?

Comment (kopiczko) [reply pkg/chartvalues/credentiald.go:21]: `AdminARN` isn't used by aws-operator. This ARN is used by `opsctl open -a cloudprovider`.

Re second part: you are right. I'll remove that. Thank you.

Comment (tfussell) [reply pkg/chartvalues/credentiald.go:21]: Got it. Thanks.

**Review by MarcelMue — APPROVED** [review]
Fine with me if it works for you.

**Comment by kopiczko** [comment]
Tested with aws-operator here https://github.com/giantswarm/aws-operator/pull/1880

**Pull request merged**